queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+/*Milestone 01 */
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+SELECT *
+FROM animals
+WHERE date_of_birth BETWEEN '01-01-2016' AND '12-12-2019';
+SELECT name
+FROM animals
+WHERE neutered = True
+    AND escape_attempts < 3;
+SELECT date_of_birth
+FROM animals
+WHERE name NOT IN ('Agumon', 'Pikachu');
+SELECT name,
+    escape_attempts
+FROM animals
+WHERE weight_kg > 10.5;
+SELECT *
+FROM animals
+WHERE neutered = true;
+SELECT *
+FROM animals
+WHERE name = 'Gabumon';
```

Comment:
- [ ] This query should find all animals **not** named Gabumon, kindly change the `=` to `<>` so it return the correct data